deps/flatgeobuf/flatgeobuf_c.h

```diff
@@ -0,0 +1,108 @@
+#ifndef FLATGEOBUF_C_H
+#define FLATGEOBUF_C_H
```

Comment:
is this coming from somewhere? I expected flatbuffers to be not tightly coupled with flatgeobuf?

Reply:
No this is the C API wrapper. I've added a banner.